pkg/controller/flow/reconcile.go

```diff
@@ -296,15 +287,6 @@ func (r *reconciler) createFeed(channelDNS string, flow *v1alpha1.Flow) (*feedsv
 	return feed, nil
 }
 
-func (r *reconciler) NewControllerRef(flow *v1alpha1.Flow) *metav1.OwnerReference {
```

Comment:
This will have a merge conflict now after https://github.com/knative/eventing/pull/391